src/components/Tables/ClientSideFilterTable.tsx

```diff
@@ -0,0 +1,249 @@
+
+      searchNodeChildren.push({
+        operation: 'and',
+        children: filterValueChildren,
+      });
+    }
+
+    // Apply filters that are outside the table but pertinent to the view (like participants or projects)
+    if (extraTableFilters) {
+      extraTableFilters.forEach((searchNodeChild: SearchNodePayload) => {
+        searchNodeChildren.push(searchNodeChild);
+      });
+    }
+
+    return {
+      operation: 'and',
+      children: searchNodeChildren,
+    };
+  }, [debouncedSearchTerm, filters, fuzzySearchColumns, extraTableFilters]);
+
+  // reset filters when extraTableFilters change
+  useEffect(() => {
+    setFilters((_filters) => {
+      const newFilters = { ..._filters };
+      Object.keys(newFilters)?.forEach((key) => {
+        const filter = {
+          ...newFilters[key],
+          values: [],
+        };
+        newFilters[key] = filter;
+      });
+      return newFilters;
+    });
+  }, [extraTableFilters]);
+
+  // set current filters if any featuredFilters has initial value, but not if we have sticky filters
+  useEffect(() => {
+    if (!sessionFilters) {
+      // Wait for session filters to finish loading
+      return;
+    }
+
+    if (stickyFilters && Object.keys(sessionFilters).length > 0) {
+      return;
+    }
+
+    const filtersWithValues = featuredFilters?.filter((ff) => ff.values && ff.values.length > 0);
+    if (filtersWithValues && filtersWithValues.length > 0) {
+      const newCurrentFilters = filtersWithValues.reduce(
+        (
+          acc: {
+            [x: string]: SearchNodePayload;
+          },
+          filter: FilterConfigWithValues
+        ): {
+          [x: string]: SearchNodePayload;
+        } => ({
+          ...acc,
+          [filter.field]: filter.searchNodeCreator
+            ? filter.searchNodeCreator(filter.values || [])
+            : defaultSearchNodeCreator(filter.field, filter.values || []),
+        }),
+        {} as {
+          [x: string]: SearchNodePayload;
+        }
+      );
+      setFilters(newCurrentFilters);
+    }
+  }, [featuredFilters, sessionFilters, stickyFilters]);
+
+  const _featuredFilters = useMemo(() => {
+    if (!featuredFilters) {
+      return undefined;
+    }
+
+    return filterModifiers ? filterModifiers(featuredFilters) : featuredFilters;
+  }, [featuredFilters, filterModifiers]);
+
+  const _filteredSortedRows = useMemo(() => {
+    const search: SearchNodePayload = getSearchPayload();
+    const numberFields = columns(activeLocale)
+      .filter((col) => col.type === 'number')
+      .map((col) => col.key);
+
+    let searchOrderConfig: SearchOrderConfig | undefined;
+    if (activeLocale && searchSortOrder) {
+      searchOrderConfig = {
+        locale: activeLocale,
+        sortOrder: searchSortOrder,
+        numberFields: numberFields,
+      };
+    }
+
+    if (searchSortOrder) {
+      return searchAndSort(rows, search, searchOrderConfig);
```

Comment:
`searchAndSort` is required for the deliverables and applications tables but optional for all others. It should be optional, defaulting to `genericSearchAndSort` when unavailable.